INTHEWILD.md

```diff
@@ -397,6 +397,7 @@ Currently, **officially** using Airflow:
 1. [Umami Collective](https://umamicollective.com) [[@juanuicich](https://github.com/juanuicich)]
 1. [United Airlines](https://www.united.com/) [[@ilopezfr](https://github.com/ilopezfr)]
 1. [Upsight](https://www.upsight.com)
+1. [US Bank](https://www.usbank.com)[@BShraman]
```

Comment:
Should be  a separate PR

Reply:
My bad, i am trying to pull back request "INTHEWILD.md" , is there is a way ??

Reply:
Rebase to latest main, amend the commit and push. If you are not familiar with rebase workflow see https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#how-to-rebase-pr

Reply:
Done  and requested ... Thank you @potiuk.